packages/central-server/src/apiV2/GETHandler/helpers.js

```diff
@@ -86,6 +86,35 @@ const getForeignKeyColumnName = foreignTable => {
   return exceptions[foreignTable] || `${foreignTable}_id`;
 };
 
+const addJoin = (
```

Comment:
This function on its own would be great to cover with unit tests, to make sure the inputs and outputs are expected in the different scenarios. It is quite dense and specific, and has well defined inputs and outputs, so is a great candidate for testing